modules/grafana/manifests/init.pp

```diff
@@ -14,9 +14,9 @@
     ensure => directory,
   }
 
-  file { '/etc/grafana/config.js':
+  file { '/etc/grafana/grafana.ini':
     ensure => file,
-    source => 'puppet:///modules/grafana/config.js',
```

Comment:
`modules/grafana/files/config.js` can be removed from this repo. You could also tidy up `/etc/grafana/config.js` on disk using `ensure => absent` so that people don't get confused when they go looking for Grafana config.